code/modules/clothing/spacesuits/rig/rig.dm

```diff
@@ -63,6 +63,7 @@
 	var/obj/item/rig_module/vision/visor                      // Kinda shitty to have a var for a module, but saves time.
 	var/obj/item/rig_module/voice/speech                      // As above.
 	var/obj/item/rig_module/storage/storage					  // Internal storage, can only have one
+	var/obj/item/rig_module/mounted/stasis/stasis			  // Stasis module
```

Comment:
stasis seems a little niche to have its own var, whats the reason for this?

Reply:
to check if you have it or not. having it allows to replace `for var/obj/item/rig_module/mounted/stasis/stasis in back.rig.modules)` with just `if(back.rig.stasis)`